### ISSUE TITLE
Add Insulin or other medications - the type, dose and time of reception

### ISSUE THREAD

**Issue by bkerensa** [Glucosio/android#74]
We should allow people to track Insulin and other diabetes medication in the app and get reminders and nags in assistant.

**Comment by HarKoss** [comment]
Hi,

It look like we can only edit glucose in the history tab. 
I saw in the HistoryFragment that OnClickListener only create AddGlucoseActivity.

can I help on that one ?

Regards,

**Comment by bkerensa** [comment]
@PaoloRotolo We still need to work on the UI for this and pull in @satya164 please have a look at Diabetes:M as a rough example of pill and insulin input we should try and fit another icon

**Comment by satya164** [comment]
![1 1-glucosio-add-types](https://cloud.githubusercontent.com/assets/1174278/17124580/ba9e3c1c-5309-11e6-8fb0-835e13af0546.png)

**Comment by bkerensa** [comment]
Yeah and we might replace insulin with "Medication" so we can have a add dialog that does both pills and insulin in one dialog versus adding YAI (Yet Another Icon)

**Comment by bkerensa** [comment]
Also on the menu we might put "Add new entry" instead of Add new reading" since medication is not a reading.

**Comment by PaoloRotolo** [comment]
Looks good! @bkerensa can you provide additional info to put in "Medication" add dialog?

**Comment by bkerensa** [comment]
Yes give me till this weekend still wrapping up family stuff then I can dedicate some time.
